examples/spring-boot-on-kubernetes-example/pom.xml

```diff
@@ -30,12 +30,6 @@
       <version>${version.spring-boot}</version>
     </dependency>
 
-    <dependency>
```

Comment:
We need this.

Reply:
Please, don't review this PR yet. It's a try and error PR, to investigate what is the root cause of the failures.

Reply:
Update PR to fix the diagnosis only.